### DISCUSSION TITLE
Traxxas TRX4M protocol

### DISCUSSION THREAD

**Discussion by vandusinae** [pascallanger/DIY-Multiprotocol-TX-Module#790] [General]
Here's a photo of the Traxxas TRX4M radio circuit. Chip is a CC2640. Is it possible to use an inexpensive 24MHz8CH analyzer to get the necessary data from that chip/circuit? 

![img_4971-x](https://user-images.githubusercontent.com/56637534/209379916-760895a7-4f52-47f4-8da8-5c12e6efa8d7.jpg)

![img_4964-x](https://user-images.githubusercontent.com/56637534/209379872-4bb0cae4-d6a5-4152-ac65-96d091e6e9ef.jpg)

**Comment by jleftbrane** [comment]
this would be great! I hope we can

**Comment by vladovg** [comment]
TX 6516 works with RX 6519

[Link](https://traxxas.com/support/Traxxas-TQi-Compatibility-Guide)


>Feb 19, 2024, 09:57 PM Pascal - Traxxas/6519 should now be working. Can someone check?
You need to compile the code by yourself because the full build doesn't fit in flash anymore...

**Comment by jleftbrane** [comment]
I'm so new to this I actually don't know how to compile and upload to my reciever. I'm happy to learn tho 💪Thank you Jleftbrane On Feb 22, 2024, at 3:30 AM, vladovg ***@***.***> wrote:﻿
TX 6516 works with RX 6519
Link

Feb 19, 2024, 09:57 PM Pascal - Traxxas/6519 should now be working. Can someone check?
You need to compile the code by yourself because the full build doesn't fit in flash anymore...

**Reply by vladovg** [reply]
[Compiling STM32](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/docs/Compiling_STM32.md)

**Comment by pascallanger** [comment]
Protocol is available Traxxas/TQ using the SFC build: https://downloads.multi-module.org/latest-test/
